boards/sim/sim/sim/configs/adb/defconfig

```diff
@@ -0,0 +1,113 @@
+#
+# This file is autogenerated: PLEASE DO NOT EDIT IT.
+#
+# You can use "make menuconfig" to make any modifications to the installed .config file.
+# You can then do "make savedefconfig" to generate a new defconfig file that includes your
+# modifications.
+#
+# CONFIG_NSH_CMDOPT_HEXDUMP is not set
+# CONFIG_SIG_SIGPIPE_ACTION is not set
+# CONFIG_SIG_SIGSTOP_ACTION is not set
+CONFIG_ADBD_AUTHENTICATION=y
+CONFIG_ADBD_AUTH_PUBKEY=y
+CONFIG_ADBD_BOARD_INIT=y
+CONFIG_ADBD_DEVICE_ID="serialno"
+CONFIG_ADBD_FILE_SERVICE=y
+CONFIG_ADBD_LOGCAT_SERVICE=y
+CONFIG_ADBD_NET_INIT=y
+CONFIG_ADBD_SHELL_SERVICE=y
+CONFIG_ADBD_TCP_SERVER=y
+CONFIG_ARCH="sim"
+CONFIG_ARCH_BOARD="sim"
+CONFIG_ARCH_BOARD_SIM=y
+CONFIG_ARCH_CHIP="sim"
+CONFIG_ARCH_SIM=y
+CONFIG_BOARDCTL_APP_SYMTAB=y
+CONFIG_BOARDCTL_POWEROFF=y
+CONFIG_BOARD_LOOPSPERMSEC=0
+CONFIG_BOOT_RUNFROMEXTSRAM=y
+CONFIG_BUILTIN=y
+CONFIG_DEBUG_ERROR=y
+CONFIG_DEBUG_FEATURES=y
+CONFIG_DEBUG_INFO=y
+CONFIG_DEBUG_NET=y
+CONFIG_DEBUG_NET_ERROR=y
+CONFIG_DEBUG_NET_WARN=y
+CONFIG_DEBUG_SYMBOLS=y
+CONFIG_DEBUG_WARN=y
+CONFIG_DEV_FIFO_SIZE=64
+CONFIG_DEV_LOOP=y
+CONFIG_DEV_PIPE_MAXSIZE=64
+CONFIG_DEV_PIPE_SIZE=64
+CONFIG_DEV_URANDOM=y
+CONFIG_DEV_ZERO=y
+CONFIG_EVENT_FD_NPOLLWAITERS=4
+CONFIG_EXAMPLES_HELLO=y
+CONFIG_EXAMPLES_HELLO_STACKSIZE=7168
+CONFIG_EXAMPLES_WEBSERVER=y
+CONFIG_EXAMPLES_WEBSERVER_DRIPADDR=0x0a000101
+CONFIG_EXAMPLES_WEBSERVER_IPADDR=0x0a000102
+CONFIG_FAT_LCNAMES=y
+CONFIG_FAT_LFN=y
+CONFIG_FSUTILS_PASSWD=y
+CONFIG_FSUTILS_PASSWD_READONLY=y
+CONFIG_FS_BINFS=y
+CONFIG_FS_FAT=y
+CONFIG_FS_PROCFS=y
+CONFIG_FS_ROMFS=y
+CONFIG_FS_TMPFS=y
+CONFIG_IDLETHREAD_STACKSIZE=4096
+CONFIG_LIBC_EXECFUNCS=y
+CONFIG_LIBUV=y
+CONFIG_LIBUV_FS_POLL=y
+CONFIG_LIBUV_NPOLLWAITERS=16
+CONFIG_LIBUV_PIPE=y
+CONFIG_LIBUV_TCP=y
+CONFIG_LIBUV_UNIT_TESTS=y
+CONFIG_LIBUV_WQ_THREADS_COUNT=2
+CONFIG_LIB_ENVPATH=y
+CONFIG_MAX_TASKS=64
+CONFIG_NET=y
+CONFIG_NETINIT_DRIPADDR=0x0a000101
+CONFIG_NETINIT_IPADDR=0x0a000102
+CONFIG_NETUTILS_PING=y
+CONFIG_NETUTILS_WEBSERVER=y
+CONFIG_NET_ICMP=y
+CONFIG_NET_LOOPBACK=y
+CONFIG_NET_LOOPBACK_PKTSIZE=64
+CONFIG_NET_PKT=y
+CONFIG_NET_SOCKOPTS=y
+CONFIG_NET_TCP=y
+CONFIG_NET_TCPBACKLOG=y
+CONFIG_NET_TCP_KEEPALIVE=y
+CONFIG_NET_UDP=y
+CONFIG_NFILE_DESCRIPTORS=32
+CONFIG_NSH_ARCHINIT=y
+CONFIG_NSH_ARCHROMFS=y
+CONFIG_NSH_BUILTIN_APPS=y
+CONFIG_NSH_FATDEVNO=2
+CONFIG_NSH_FILE_APPS=y
+CONFIG_NSH_MOTD=y
+CONFIG_NSH_MOTD_STRING="MOTD: username=admin password=Administrator"
+CONFIG_NSH_READLINE=y
+CONFIG_NSH_ROMFSDEVNO=1
+CONFIG_NSH_ROMFSETC=y
+CONFIG_PATH_INITIAL="/bin"
+CONFIG_PIPES=y
+CONFIG_POSIX_SPAWN_PROXY_STACKSIZE=2048
+CONFIG_RAMLOG=y
+CONFIG_RAMLOG_BUFSIZE=8192
+CONFIG_RAMLOG_SYSLOG=y
+CONFIG_READLINE_TABCOMPLETION=y
+CONFIG_SCHED_HAVE_PARENT=y
+CONFIG_SCHED_ONEXIT=y
+CONFIG_SDCLONE_DISABLE=y
+CONFIG_SIG_DEFAULT=y
+CONFIG_SIM_M32=y
```

Comment:
@spiriou Can you try not setting this?  Usually we do not do a 32bit build unless it is required, it also will not build on macOS.  I think this might actually resolve the issue since I dont think 32bit libz is installed in the build container.

Reply:
Done. Let's see how it goes.

Reply:
Yeah I'll keep an eye on it.  Unfortunately it is hard to do one-off testing in the CI we have it set to fail fast because otherwise we clog up the works, but that is less desirable.  GitHub supports a feature now where we can trigger a build with parameters, which might be useful to do thigs like running in verbose mode / not failing fast / selecting a single configuration.

Reply:
@spiriou If you do want to trigger a build that will not fail fast you can disable it in your PR, we just need to make sure that does not get merged.

https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast